### PR TITLE
Fixed issue with some ship placement clicks being unresponsive

### PR DESCRIPTION
**Pull request by xMestas**

### CODE REVIEW

**Comment by xMestas** [comment]
Fixes #22

File: src/main/java/cs361/battleships/models/Game.java [hunk 1]
```diff
@@ -39,7 +41,7 @@ public boolean placeShip(Ship ship, int x, char y, boolean isVertical) {
 	 */
     public boolean attack(int x, char y) {
         Result playerAttack = opponentsBoard.attack(x, y);
-        if (playerAttack.getResult() == INVALID) {
+        if (playerAttack.getResult() == AtackStatus.INVALID) {
```
[the remark this answers is not in the document]
Comment (xMestas) [reply src/main/java/cs361/battleships/models/Game.java]: Yes, that is correct.  Rand didn't even need to be there tbh but I was lazy and never changed it.  Both implementations should work though

**Comment by xMestas** [comment]
implements #22

File: src/main/java/cs361/battleships/models/Game.java [hunk 2]
```diff
@@ -71,20 +73,22 @@ else if (rand == 3) {
...
         return (char)rand;
     }
 
     public boolean randVertical() {
         // generate int between 0 and 1 inclusive
-        int rand = ThreadLocalRandom.current().nextInt(0, 2);
+        int rand = Math.abs(r.nextInt()) % 2;
 
         if (rand == 0) {
             return false;
```
Comment (lkthang) [inline src/main/java/cs361/battleships/models/Game.java]: Random functions look good!

**Pull request merged**